testing/e2e/app/routes/ComboBox/spec.ts

```diff
@@ -1,181 +1,382 @@
-import { test, expect } from '@playwright/test';
+import { test, expect, Page } from '@playwright/test';
+
+const defaultOptions = [
+  { label: 'Item 0', value: 0 },
+  { label: 'Item 1', value: 1, subLabel: 'sub label' },
+  { label: 'Item 2', value: 2 },
+  { label: 'Item 3', value: 3 },
+  { label: 'Item 4', value: 4 },
+  { label: 'Item 10', value: 10 },
+  { label: 'Item 11', value: 11 },
+];
 
 test.describe('ComboBox', () => {
-  test('should support keyboard navigation when virtualization is enabled', async ({
-    page,
-  }) => {
-    await page.goto('/ComboBox?virtualization=true');
-
-    await page.keyboard.press('Tab');
-    const comboBoxInput = page.locator('#test-component').locator('input');
-    const comboBoxMenu = page.getByRole('listbox');
-    await expect(comboBoxInput).toHaveAttribute(
-      'aria-controls',
-      'test-component-list',
-    );
-    await expect(comboBoxMenu).toBeVisible();
+  test.describe('General', () => {
+    test('should select multiple options', async ({ page }) => {
+      await page.goto('/ComboBox?multiple=true');
 
-    const items = page.getByRole('option');
+      await page.keyboard.press('Tab');
 
-    //focus first
-    await page.keyboard.press('ArrowDown');
-    await expect(comboBoxInput).toHaveAttribute(
-      'aria-activedescendant',
-      'test-component-option-Item-0',
-    );
-    await expect(items.first()).toHaveAttribute('data-iui-focused', 'true');
+      const options = await page.locator('[role="option"]').all();
+      for (const option of options) {
+        await option.click();
+      }
 
-    //stay on first
-    await page.keyboard.press('ArrowUp');
-    await expect(comboBoxInput).toHaveAttribute(
-      'aria-activedescendant',
-      'test-component-option-Item-0',
-    );
-    await expect(items.first()).toHaveAttribute('data-iui-focused', 'true');
+      const tags = await page
+        .locator('#test-component-selected-live > span')
```

Comment:
this locator is relying on an implementation detail which could change at any time?

Reply:
The `select-tag` is still coming from an internal class which can change at any time.

Right now we don't have a way to pass classes/attributes to the tag container or the tags (#894). This test could probably be rewritten to avoid relying on such implementation details (e.g. by only comparing the text inside the main container), but I'll leave it upto you if you want to rewrite it or add a TODO to wait until we allow customizing the tag container.